Sources/InstantSearch/Numeric Comparator/NumericStepperController.swift

```diff
@@ -27,6 +27,10 @@ public class NumericStepperController: NumberController {
     stepper.addTarget(self, action: #selector(stepperOnValueChanged), for: .valueChanged)
   }
 
+  public func setBounds(bounds: ClosedRange<Double>?) {
```

Comment:
This function is empty, is it expected? if yes, maybe we can put a comment inside for clarity

Reply:
Nice catch! Fixed ✔️